### ISSUE TITLE
MAL ID

### ISSUE THREAD

**Issue by Vesnyx** [DakuTree/manga-tracker#93]
Seems like the way that the site currently tracks MAL ID links is through user submission. However, it also allows anyone to edit the existing entry, which might make for some unfortunate circumstances if someone decides to have some fun with it. Perhaps crosschecking the name of the manga with the alternate names and the original name according to the MAL page might be helpful here. 

On another note, it would be nice to be able to search through MAL list results on the same page, going to the website and finding the ID via the URL is rather tedious.

**Comment by DakuTree** [comment]
> Seems like the way that the site currently tracks MAL ID links is through user submission. However, it also allows anyone to edit the existing entry, which might make for some unfortunate circumstances if someone decides to have some fun with it.

When you set the MAL id it's only set for you, not everybody else. The exception of this is ones I've manually confirmed myself, which are set for everybody.  
If you try and change the MAL id of one that is already set by default, your ID will be used instead (but only for you).

**Comment by Vesnyx** [comment]
Are you sure? I get [this ](http://i.imgur.com/G1ONlf2.png)issue when I try to change an ID that I previously set. I suppose you can check on the backend, but I first set the MAL ID for Amagi Brilliant Park to it's novel version. Then, I changed it to the manga version, pulling up this alert notification. If the issue is just a typo, then this issue is partially completed I guess? 

Another way to search and verify the ID's would be a nice enhancement.

**Comment by DakuTree** [comment]
> Are you sure? I get this issue when I try to change an ID that I previously set.  

Ahh I see what the issue is. That alert should have only been showing if we already had a MAL id set, but seems like it also showed if none was set at all and it was changed twice.  
Was just a UI bug so this was only changing your MAL ID, and not everybody elses.

It's fixed in the next update, which should be out shortly.

> Another way to search and verify the ID's would be a nice enhancement.

Problem with trying to do this is the official MAL API requires username/password to work (regardless if you are logged in on MAL), and I'd rather not get to scraping their search pages.  
Might just add some link to the MAL search if it's missing an ID which should at least save some time when trying to find the ID. 

On a somewhat related note, I went ahead and verified a few more series on the series backend (including some I saw missing on your list) so they should properly have a MAL ID now.
